stun/packet.go

```diff
@@ -41,21 +42,26 @@ func newPacket() (*packet, error) {
 }
 
 func newPacketFromBytes(packetBytes []byte) (*packet, error) {
-	if len(packetBytes) < 24 {
+	if len(packetBytes) < 20 {
```

Comment:
Is `24` valid? do we always need type and length? which requires another 4 bytes